graftm/run.py

```diff
@@ -367,26 +363,24 @@ def graft(self):
                                self.args.output_directory,
                                False)
 
+
+        # Classification steps        
```

Comment:
No, diamond is a little trickier and I don't really want to go into that part of the code just yet tbh. I just needed the clustering in the HMM side of things for timing.


I've moved the code block into the if statement.


________________________________
From: Ben J Woodcroft <notifications@github.com>
Sent: Wednesday, 26 August 2015 11:46 PM
To: geronimp/graftM
Cc: Mr Joel Boyd
Subject: Re: [graftM] Cluster now default, simplified code (#101)


In graftm/run.py<https://github.com/geronimp/graftM/pull/101#discussion_r37982645>:

> @@ -367,26 +363,24 @@ def graft(self):
>                                 self.args.output_directory,
>                                 False)
>
> +
> +        # Classification steps


shouldn't this be inside the next if statement? Or so you want to cluster before diamond as well?

-
Reply to this email directly or view it on GitHub<https://github.com/geronimp/graftM/pull/101/files#r37982645>.